yamui.c

```diff
@@ -1547,7 +1550,7 @@ main(int argc, char *argv[])
 	 * thus potentially eliminate / shorten black screen time
 	 * during compositor handover).
 	 */
-	if (debugging) {
+	if (!skip_cleanup) {
```

Comment:
nit / matter of taste: in if(do_cleanup) vs if(!skip_cleanup) I'd prefer the arrangement that does not involve "double negation." But does not really matter...

Reply:
I guess it is cleaner that way. Changed.